doc/sections/commands.md

```diff
@@ -84,6 +86,22 @@ We recommend to start types and predicates by a capital letter.
 
 These modifiers are used to help the unification engine.
 
+**Exposition marker**
+Exposition defines how a symbol can be used outside the module where it is
+defined. By default any symbol is _public_, which means it can be used without
+restriction anywhere. There are two exposition markers available:
+- `private`: the symbol cannot be used outside of its module (can be compared to
+  a symbol not in the interface in OCaml);
+- `protected`: the symbol can only be used in left-hand side of rewrite rules
+  outside of its module.
+
+Exposition obeys the following rules
+- a public symbol can't by typed by private symbols
+- a public symbol can't rewrite to private symbols
+- foreign protected symbols can be used in the left-hand side of rewrite rules
+  exclusively
+- rewrite rules can't be defined on foreign private symbols.
+
```

Comment:
Should be clarified. See https://github.com/Deducteam/lambdapi/pull/262#issuecomment-553010109 . Start by saying "Inside a module, ..."